Sources/Extensions/UIWindow+Deboogger.swift

```diff
@@ -0,0 +1,31 @@
+//
+// Copyright (c) 2019 Rosberry. All rights reserved.
+//
+
+import UIKit
+
+extension UIWindow {
+
+    private enum Constants {
+        static let numberOfTouches = 4
+        static let numberOfTaps = 2
+    }
+
+    @objc private func setupGestureRecognizer() {
+        #if targetEnvironment(simulator)
+            return
+        #endif
+
+        let recognizer = UITapGestureRecognizer(target: self, action: #selector(showDeboogger))
+        recognizer.numberOfTapsRequired = Constants.numberOfTaps
```

Comment:
Hm... I have an idea:
```swift
let recognizer = UITapGestureRecognizer(target: Deboogger.shared, action: #selector(show))
```
I'm not sure that it's a good code... What do you think?

Reply:
This isn't good idea because `show()` function of Deboogger should become `@objc` ☝️🤓